lms/djangoapps/ora_staff_grader/utils.py

```diff
@@ -43,7 +43,16 @@ def wrapped_function(
     return decorator
 
 
-def call_xblock_json_handler(request, usage_id, handler_name, data):
+def is_json(input_string):
+    """Quick True/False check to see if a value is JSON"""
+    try:
+        json.loads(input_string)
```

Comment:
I'm confused here. We take the effort to generate the JSON object (of unknown source length) and throw away the results?  Rationel for throwing it away?

Reply:
This was to keep the XBlock handler code cleaner. We regularly need to do inline checking of "is the returned payload HTML or JSON?" and conditionally unpack based off the result. We could add this `try/except` in the handler APIs to save some cycles but at a cost of code readability and cleanliness

Reply:
Meh. Code readability is important, so leaving this as-is is fine